### ISSUE TITLE
External provider only option

### ISSUE THREAD

**Issue by fgallaire** [netlify/netlify-identity-widget#82]
It could be useful to have an external provider only option, to remove the "cumbersome" Log in/Sign up UI not needed.

**Comment by calavera** [comment]
I agree, I opened an issue in the backend to discuss how to do that: https://github.com/netlify/gotrue/issues/109
